R/summarize.R

```diff
@@ -88,6 +85,12 @@ summarize_over_time <- function (data_filtered) {
       sum_cap100k_new_deaths   = sum_discarding_incomplete(all_new_deaths / pop_100k),
       cap100k_new_tests    = mean_discarding_incomplete(all_new_tests / pop_100k),
       sum_cap100k_new_tests    = sum_discarding_incomplete(all_new_tests / pop_100k),
+      
+      # relocate here so as not to complicate the results
+      # this helps to avoid having all sums and means to be same numbers
+      all_new_cases            = mean_discarding_incomplete(all_new_cases),
```

Comment:
Yes, thanks, this was probably necessary. Before, in line 70, we calculated:
```
sum_all_new_cases        = sum_discarding_incomplete(mean_discarding_incomplete(all_new_cases))
```

The problem is that dplyr uses the variables generated in previous steps, as discussed with Antoine here:

https://app.clickup.com/t/28ffh75